src/builder.rs

```diff
@@ -156,6 +151,24 @@ impl DoubleArrayAhoCorasickBuilder {
         self
     }
 
+    /// Specifies the number of last blocks to search bases.
+    ///
+    /// When the RAM capacity is small, this should be a small value.
```

Comment:
```suggestion
    /// The smaller the number is, the faster the construction time will be;
    /// however, the memory efficiency can be degraded.
```

When the number is small, the search area is limited, and the number of empty elements can increase.
Therefore, the memory of `extras` becomes always smaller, but the total memory does not become always smaller.